package.json

```diff
@@ -25,8 +25,9 @@
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
     "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
The error mentioned adding it to the `devDependencies` but i'm sure it works here too
e.g. `yarn add --dev @babel/plugin-proposal-private-property-in-object`

The PR looks good to me otherwise tbh. There's no breaking change.

Reply:
Fixed. Thanks @Obsiye.